meta/meta_autoid.go

```diff
@@ -132,8 +144,13 @@ func (a *autoIDAccessors) RowID() AutoIDAccessor {
 }
 
 // IncrementID is used to get the auto_increment ID meta key-value accessor.
-func (a *autoIDAccessors) IncrementID() AutoIDAccessor {
-	a.access.idEncodeFn = a.access.m.autoIncrementIDKey
+func (a *autoIDAccessors) IncrementID(tableVersion uint16) AutoIDAccessor {
+	// _tidb_rowid and auto_increment ID in old version TiDB share the same meta key-value.
+	if tableVersion < sepAutoIncVer {
```

Comment:
This version of `sepAutoIncVer` is not available yet?

Reply:
Yea, so the value is `model.TableInfoVersion4 + 1`. Later I will update it to `model.TableInfoVersion5`